lib/app.jsx

```diff
@@ -508,7 +509,25 @@ export const App = connect( mapStateToProps, mapDispatchToProps )( React.createC
 				}
 				{ isAuthorized ?
 						<div className={mainClasses}>
-							{ state.showNavigation &&
+							<div className="sidebar">
+								<div className="top-bar theme-color-border">
+									<ChangeLayout />
```

Comment:
I think we could improve the naming. This reads like a verb when I think it's probably more akin to a noun such as "LayoutChanger" or "LayoutSelection" or "VisibilityToggles"